int("3.5")
# 出力結果　ValueError
